app/models/rdv.rb

```diff
@@ -85,11 +97,11 @@ def temporal_status
   end
 
   def possible_temporal_statuses
-    if in_the_future?
-      ["unknown_future", starts_at.to_date.today? ? "waiting" : nil, "excused"].compact
-    else
-      %w[unknown_past seen notexcused excused]
-    end
+    return %w[unknown_past seen notexcused excused] if in_the_past?
+    return %w[unknown_future seen waiting excused] if in_one_hour?
```

Comment:
Ahah, et forcément, le test échoue suite à ma suggestion 🙈 
```
     NoMethodError:
       undefined method `in_one_hour?' for #<Rdv:0x0000558a63616c58>
       Did you mean?  in_next_hour?
```

```suggestion
    return %w[unknown_future seen waiting excused] if in_next_hour?
```

Reply:
et moi qui n'ais pas vérifié correctement :sweat_smile: